README.md

```diff
@@ -57,7 +57,7 @@ _Pst. hey, you, give us a star if you like it!_
 
 ### Requirements
 
-1. An **[Arcade AI account](https://arcade-ai.typeform.com/early-access)** (currently a waitlist)
+1. An **[Arcade account](https://arcade-ai.typeform.com/early-access)** (currently a waitlist)
```

Comment:
Update to the sign up link we have in the docs: https://account.arcade.dev/registerOrRedirect?return_to=https%3A%2F%2Fapi.arcade.dev%2Fdashboard&new_user_return_to=https%3A%2F%2Fapi.arcade.dev%2Fdashboard%2Fwelcome

No longer waitlisted!

Reply:
Good catch - updated!